vtk_viewer/cmake/vtk_viewer-config.cmake.in

```diff
@@ -0,0 +1,13 @@
+@PACKAGE_INIT@
+
+set(@PROJECT_NAME@_FOUND ON)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
+set_and_check(@PROJECT_NAME@_LIBRARIES "${PACKAGE_PREFIX_DIR}/lib")
+
+include(CMakeFindDependencyMacro)
+find_dependency(cmake_common_scripts)
+find_dependency(console_bridge)
+find_dependency(PCL 1.9)
+find_dependency(VTK 7.1)
```

Comment:
I don't believe you need the version number here. I think it should pull the information for the find package in the CMakeLists.txt file which is the nice thing about using find_dependency.

Reply:
Addressed